README.md

```diff
@@ -2,23 +2,163 @@
 
 A minimal GameCube IPL
 
+IPLBoot will attempt to load DOLs from the following locations in order:
+- USB Gecko in Card Slot B
+- SD Gecko in Card Slot B
+- USB Gecko in Card Slot A
+- SD Gecko in Card Slot A
+- SD2SP2
+
+When loading from SD Gecko or SD2SP2, it will look for and load different filenames depending on what buttons are being held at boot. You can use this to change the boot behavior by holding a button on your controller. 
+
+ Button Held | File Loaded
+-------------|--------------
+ *None*      | `/ipl.dol`
+ A           | `/a.dol`
+ B           | `/b.dol`
+ X           | `/x.dol`
+ Y           | `/y.dol`
+ Z           | `/z.dol`
+ Start       | `/start.dol`
+ D-Left      | `/left.dol`
+ D-Right     | `/right.dol`
+ D-Up        | `/up.dol`
+
+If no DOL can be loaded, IPLBoot will boot the official IPL (GameCube intro and menu) instead.
+
+For example, this configuration would boot straight into Swiss by default, or GBI if you held B, or the typical GameCube intro if you held any other button:
+- `/ipl.dol` - Swiss
+- `/b.dol` - GBI
+
+This configuration would boot into the typical GameCube intro by default, or Swiss if you held Z, or GBI if you held B:
+- `/z.dol` - Swiss
+- `/b.dol` - GBI
+
+**Pro-tip:** You can prevent files from showing in Swiss by marking them as hidden files on the SD card.
+
+If you hold multiple buttons, the highest in the table takes priority. Also note that all directions are D-Pad buttons, *not* joysticks. Careful not to touch any of the analog controls (sticks and triggers) when powering on as this is when they are calibrated.
+
+**Something not working?** See the [troubleshooting section](#troubleshooting).
+
+
 ## Usage
 
-Flash the latest build to your Qoob Pro or Viper as a BIOS.  
-The Qoob SX is currently not supported because it uses a very different process
-to boot, and reverse-engineering efforts so far have been unsuccessful.
+Download the designated appropriate file for your device from the [latest release](https://github.com/redolution/iplboot/releases/latest).
+
+Prepare your SD card by copying DOLs onto the SD card and renaming them according the table above.
+
+### PicoBoot
+
+You can download ready-to-go firmware featuring IPLBoot directly from PicoBoot. Just follow the [installation guide](https://github.com/webhdx/PicoBoot#-installation-guide). This is likely what you want.
```

Comment:
The guide that is linked to includes instructions on where and what to download. I'd rather PicoBoot maintain that.

Reply:
The linked installation guide has instructions on where and what to download. I'd rather not maintain that info here.

Reply:
I'm confused. I was suggesting linking to the PicoBoot repo, not directly to the download. That's completely optional though since you already link the installation instructions.

Reply:
Ah I see. I thought you meant linking directly to the firmware file. Will do.